code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -85,4 +86,4 @@
 	sprites = list(
 		"Gravekeeper" = "cogspider"
 		)
-	law_type = "Gravekeeper"
```

Comment:
I think an indentation was removed here

Reply:
Does look like that happened somehow. Fixing.

Reply:
still broken btw, ya didn't fix it